src/isomedia/box_code_base.c

```diff
@@ -24,6 +24,7 @@
  */
 
 #include <gpac/internal/isomedia_dev.h>
+#include <gpac/internal/media_dev.h>
```

Comment:
@jeanlf Should we need this? or should some code from this PR be moved?

Reply:
no we shouldn't this comes from gf_av1_leb128_size I guess

Reply:
`iacb_box_size()` now calls `gf_odf_ia_cfg_size()` instead, which removes the need for using `gf_av1_leb128_size` and having this dependency on media_dev.h